web/webkit/src/main/scala/net/liftweb/http/provider/HTTPProvider.scala

```diff
@@ -132,6 +132,9 @@ trait HTTPProvider {
   }
 
   private def postBoot {
+    if (!LiftRules.logServiceRequestTiming) {
+      LiftRules.installServiceRequestTimer(NoOpServiceTimer)
+    }
```

Comment:
I'm not sure I quite understand this line. If request timing is off, we install the no-op timer in the new place?

Reply:
What I'm proposing is to remove ```logServiceRequestTiming``` and only use ```ServiceRequestTimer```, but we cannot make such a breaking change (removing```logServiceRequestTiming```) in 3.x can we? So to not break existing apps the line in ```postBoot``` installs the ```NoOpServiceTimer``` implementation which will give the same behavior as today's ```logServiceRequestTiming=false```. Currently ```logServiceRequestTiming``` defaults to ```true```, which corresponds to having ```StandardServiceTimer``` as default, which is the default in this PR.